src/resources/routes.json

```diff
@@ -0,0 +1,27 @@
+{
+  "GETSTATUS": {
```

Comment:
we need to name these keys properly. ex: `LIST_DATASOURCES` instead of "LISTDATASOURCES"